pkg/ddc/alluxio/transform_api_gateway.go

```diff
@@ -0,0 +1,18 @@
+package alluxio
+
+import (
+	"strconv"
+
+	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
+)
+
+// transformAPIGateway transforms the given value
+func (e *AlluxioEngine) transformAPIGateway(runtime *datav1alpha1.AlluxioRuntime, value *Alluxio) (err error) {
+
+	if runtime.Spec.APIGateway.Enabled {
```

Comment:
Is it a redundancy here?  It seems the property is already set in `setPortProperties` func

Reply:
Yes, you are correct.